go.mod

```diff
@@ -3,83 +3,57 @@ module github.com/jaegertracing/jaeger
 go 1.14
 
 require (
-	github.com/AndreasBriese/bbloom v0.0.0-20190825152654-46b345b51c96 // indirect
```

Comment:
I do not believe this PR requires changes to dependencies. Please exclude the go.mod/go.sum files.

Reply:
I've synced my fork with the upstream master. Do I still need to exclude go.mod/go.sum files?

Reply:
if this PR does not require new dependencies, it should not be updating dependency file. Please do `checkout master <file>` to get the originals.